src/frontend/plugins/gatsby-plugin-top-layout/TopLayout.js

```diff
@@ -1,10 +1,44 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+import { red } from '@material-ui/core/colors';
 import CssBaseline from '@material-ui/core/CssBaseline';
-import { ThemeProvider } from '@material-ui/core/styles';
-import theme from '../../src/theme';
+import PropTypes from 'prop-types';
 
 export default function TopLayout(props) {
+  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
+
+  // Color Overrides
+  const primaryMainColor = prefersDarkMode ? '#303030' : '#333E64';
+  const secondaryMainColor = prefersDarkMode ? '#4f96d8' : '#0589D6';
+  const backgroundMainColor = prefersDarkMode ? '#121212' : '#E5E5E5';
+  const textDefaultColor = prefersDarkMode ? '#e5e5e5' : '#181818';
+  const textVisitedLinkColor = prefersDarkMode ? 'pink' : 'purple';
+
+  const theme = createMuiTheme({
+    palette: {
+      type: prefersDarkMode ? 'dark' : 'light',
+      primary: {
+        main: primaryMainColor,
+      },
+      secondary: {
+        main: secondaryMainColor,
+      },
+      error: {
+        main: red.A400,
+      },
+      background: {
+        default: backgroundMainColor,
+      },
+      text: {
+        primary: '#E5E5E5',
+        secondary: secondaryMainColor,
+        visited: textVisitedLinkColor,
+        default: textDefaultColor,
+      },
+    },
+  });
+
   return (
```

Comment:
@humphd : This is great work but I am afraid that it will create more work when we port to Next. To minimize the amount of work in Next, the easiest way is to follow MUI theming structure which was what i did in #1397 .  
![image](https://user-images.githubusercontent.com/55034244/99472026-c6207500-2915-11eb-8cf8-653eaca28424.png)

Reply:
the `text` property again will be a problem when we port to NextJS

Reply:
There is a workaround that allows you to kinda overwrite default MUI property. In that case, you can do whatever structure you like. The main culprit here is TypeScript. It is quite strict.

Reply:
@abhaseen